src/constants.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Environment } from "@azure/ms-rest-azure-env";
+import { localize } from "./utils/localize";
+
```

Comment:
Just make sure we're not throwing every single constant in here. IMO this file should only be used for constants shared across multiple files or "noteworthy" constants